### PR TITLE
Apply tiny polish to RESTEasy Reactive startup code

### PR DESCRIPTION
**Pull request by geoand**

### CODE REVIEW

File: independent-projects/resteasy-reactive/server/runtime/src/main/java/org/jboss/resteasy/reactive/server/core/startup/RuntimeDeploymentManager.java [hunk 1]
```diff
@@ -180,8 +180,9 @@ public Deployment deploy() {
 
         //pre matching interceptors are run first
         List<ServerRestHandler> preMatchHandlers = new ArrayList<>();
-        for (HandlerChainCustomizer customizer : info.getGlobalHandlerCustomers()) {
-            preMatchHandlers.addAll(customizer.handlers(HandlerChainCustomizer.Phase.BEFORE_PRE_MATCH));
+        for (int i = 0; i < info.getGlobalHandlerCustomers().size(); i++) {
```
Comment (gsmet) [inline independent-projects/resteasy-reactive/server/runtime/src/main/java/org/jboss/resteasy/reactive/server/core/startup/RuntimeDeploymentManager.java:183]: Hmmm, customer or customizer?

**Review by gsmet — DISMISSED** [review]
Spotted a small (but preexisting) inconsistency.

File: independent-projects/resteasy-reactive/server/runtime/src/main/java/org/jboss/resteasy/reactive/server/core/startup/RuntimeDeploymentManager.java (hunk 1 above)
Comment (geoand) [reply independent-projects/resteasy-reactive/server/runtime/src/main/java/org/jboss/resteasy/reactive/server/core/startup/RuntimeDeploymentManager.java]: Good eye! Fixed

**Comment by geoand** [comment]
@gsmet mind giving this a quick review since you already took a look at it?

**Review by gsmet — APPROVED** [review]

**Pull request merged**